### DISCUSSION TITLE
Select a specific version while using CDN

### DISCUSSION THREAD

**Discussion by Yaronglp** [rsms/inter#644] [Q&A] [answered]
Hello,

I'd appreciate it if you could let me know whether there is an option to utilize the CDN feature while also providing parameters to obtain a particular version. 
For instance, I'm interested in using version 3.19, so would the correct approach be to use this URL: https://rsms.me/inter/inter.css?v=3.19?

**Comment by doutatsu** [comment]
Same here - some users reporting major issues with the font (extremely thin) on my site, and timing seems to match the release of v4. So I want to switch to 3 to see if it fixes this issue

<img width="644" alt="image" src="https://github.com/rsms/inter/assets/4270980/e892f887-d25b-4b03-9c12-c536449e0bec">

**Comment by Yaronglp** [comment]
Hi @rsms,

Do you support versions within the CDN you are utilizing?

10x :)

**Comment by rsms** [comment, marked as answer]
Short answer: To use a specific version, host the font files yourself ([download whatever version you want here](https://github.com/rsms/inter/releases))

Interesting idea. I wonder what the use case is for using an older version? (with the one exception of 3.19 which was quite different from v4.)

For simplicity and cost reasons, only the current version is stored on the CDN.
As mentioned above, the preferred method for someone who wants control is to host whatever font files they wants themselves. This is also how you can apply subsetting to shave off a few kB.

**Reply by Yaronglp** [reply]
10x for the response :)
